### DISCUSSION TITLE
GrapesJS icons plugin with Iconify

### DISCUSSION THREAD

**Discussion by bgrand-ch** [GrapesJS/grapesjs#5259] [Show and tell]
```
Disclaimer: Work in progress!
```

Hello,

I need to easily add icons in a GrapesJS page for a project.

I created the following plugin using the amazing [Iconify framework](https://github.com/iconify):

> https://github.com/bgrand-ch/grapesjs-icons

If you are interested, could you test it locally and give me a feedback please?

CC @artf

**Comment by artf** [comment]
Looks awesome 😍

**Comment by bgrand-ch** [comment]
🆕 New version with better performance, better UI/UX and better docs!
